src/pages/VerificationPage/VerificationPage.jsx

```diff
@@ -0,0 +1,24 @@
+import VerificationForm from 'components/VerificationForm/VerificationForm'
+import * as styleMui from './VerificationPage.styled'
+import { motion } from 'framer-motion'
+
+function VerificationPage() {
+    return (
+        <styleMui.container>
+            <styleMui.Background>
+                <motion.div
+                    initial={{ opacity: 0, x: '-30%' }}
+                    animate={{ opacity: 1, x: '0%' }}
+                    exit={{ opacity: 0, x: '30%' }}
+                    transition={{ duration: 0.5, ease: 'easeOut' }}
+                >
+                    <styleMui.formContainer>
+                        <VerificationForm />
+                    </styleMui.formContainer>
+                </motion.div>
```

Comment:
You should use component in MUI for this tag
Reading here: [How to use framer-motion with MUI?](https://stackoverflow.com/a/65681968)